hack/verify-vendor-licenses.sh

```diff
@@ -33,16 +33,14 @@ function cleanup {
 }
 kube::util::trap_add cleanup EXIT
 
-cp -r "${KUBE_ROOT}/Godeps" "${_tmpdir}/Godeps"
-ln -s "${KUBE_ROOT}/LICENSE" "${_tmpdir}"
 ln -s "${KUBE_ROOT}/vendor" "${_tmpdir}"
 ln -s "${KUBE_ROOT}/staging" "${_tmpdir}"
 
 # Update vendor Licenses
 LICENSE_ROOT="${_tmpdir}" "${KUBE_ROOT}/hack/update-vendor-licenses.sh"
 
 # Compare vendor Licenses
-if ! _out="$(diff -Naupr "${KUBE_ROOT}/Godeps/LICENSES" "${_tmpdir}/Godeps/LICENSES")"; then
+if ! _out="$(diff -Naupr "${KUBE_ROOT}/vendor/ALL_LICENSES" "${_tmpdir}/vendor/ALL_LICENSES")"; then
```

Comment:
we're symlinking the vendor dir... which means when we regenerate ALL_LICENSES, we're modifying the original vendor dir, which means this diff will never show a difference, even if ALL_LICENSES was not originally correct

Reply:
Thank you for you review comments. I updated the code diff. Thanks.